schemas/linkedDataSignature2020.js

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) 2012-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const w3cDateTime = require('./w3cDateTime');
+const identifier = require('./identifier');
+
+const baseSignature = {
+  title: 'Linked Data Signature',
+  description: 'A Linked Data digital signature.',
+  // NOTE: id is not required
+  required: ['type', 'created', 'proofValue'],
```

Comment:
Why isn't `verificationMethod` required?

Reply:
Fixed in be0dcee8b742ff4328dc606c3215d5c01d85177e.